<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewBookInfo</class>
 <widget class="QWidget" name="ViewBookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QPushButton{
	padding: 8px 20px;
	border-radius: 5px;
	outline: none;
	color: rgb(0, 0, 0);
	font: 700 9pt &quot;Segoe UI&quot;;
	background-color: rgb(192, 220, 161);
	
}

QPushButton:hover {
	background-color: rgb(161, 184, 135);
}


QLabel{
	color: rgb(0, 0, 0);
	font: 700 9pt &quot;Segoe UI&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bannerBackground" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#bannerBackground {
	background-image: url(:/branding/images/brand/background.jpg);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,0">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="bannerLogo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bannerTitle">
        <property name="styleSheet">
         <string notr="true">#bannerTitle {
	color: rgb(255, 255, 255);
	font: 700 60pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>City Library</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="bookInfoScroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1215</width>
        <height>586</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#bookInfoScroll{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bookLayout" native="true">
         <property name="styleSheet">
          <string notr="true">#bookLayout{
	border: 3px solid;
	border-color: rgb(158, 206, 104);
	background-color: rgb(227, 238, 219);
}</string>
         </property>
         <layout class="QHBoxLayout" name="bookInfoLayout">
          <item>
           <widget class="QWidget" name="bookCoverLayout" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="coverLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="bookDescriptionLayout" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="titleLabel">
               <property name="styleSheet">
                <string notr="true">#titleLabel{
	font: 700 26pt &quot;Segoe UI&quot;;
}</string>
               </property>
               <property name="text">
                <string>title</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="detailsLayout" stretch="0">
               <item>
                <widget class="QWidget" name="info" native="true">
                 <property name="styleSheet">
                  <string notr="true">#info{
	border: 0px solid;
}</string>
                 </property>
                 <layout class="QGridLayout" name="bookDetailsLayout" columnstretch="0,1">
                  <item row="0" column="0">
                   <widget class="QLabel" name="authorLabel">
                    <property name="text">
                     <string>Author:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="isbnLabel">
                    <property name="text">
                     <string>ISBN:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="isbnOutputLabel">
                    <property name="text">
                     <string>placeHolder</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="authorOutputLabel">
                    <property name="text">
                     <string>placeHolder</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QTextBrowser" name="bookDescription"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QStackedWidget" name="statusStackedWidget">
                    <property name="styleSheet">
                     <string notr="true">#statusStackedWidget{
	
	background-color: rgb(227, 238, 219);
}
</string>
                    </property>
                    <property name="currentIndex">
                     <number>1</number>
                    </property>
                    <widget class="QWidget" name="notAvailablePage">
                     <property name="styleSheet">
                      <string notr="true">#notAvailablePage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_8">
                      <item>
                       <widget class="QLabel" name="notAvailableLabel">
                        <property name="styleSheet">
                         <string notr="true">#notAvailableLabel{
	color: rgb(255, 0, 0);
	font: 700 20pt &quot;Segoe UI&quot;;
}</string>
                        </property>
                        <property name="text">
                         <string>Not Available</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="availablePage">
                     <property name="styleSheet">
                      <string notr="true">#availablePage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_7">
                      <item>
                       <widget class="QLabel" name="availableLabel">
                        <property name="styleSheet">
                         <string notr="true">#availableLabel{
	color: rgb(0, 170, 0);
	font: 700 20pt &quot;Segoe UI&quot;;
}</string>
                        </property>
                        <property name="text">
                         <string>Available</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="checkedoutPage">
                     <property name="styleSheet">
                      <string notr="true">#checkedoutPage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_9">
                      <item>
                       <widget class="QLabel" name="checkedoutLabel">
                        <property name="styleSheet">
                         <string notr="true">#checkedoutLabel{
	color: rgb(0, 170, 0);
	font: 700 20pt &quot;Segoe UI&quot;;
}</string>
                        </property>
                        <property name="text">
                         <string>Checked out to you</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <widget class="QStackedWidget" name="optionsStackedWidget">
                    <property name="currentIndex">
                     <number>1</number>
                    </property>
                    <widget class="QWidget" name="dueDatePage">
                     <property name="styleSheet">
                      <string notr="true">#dueDatePage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <item>
                       <widget class="QLabel" name="dueLabel">
                        <property name="styleSheet">
                         <string notr="true">#dueLabel{
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Segoe UI&quot;;
}</string>
                        </property>
                        <property name="text">
                         <string>Due Date :</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="dueOutputLabel">
                        <property name="styleSheet">
                         <string notr="true">#dueOutputLabel{
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Segoe UI&quot;;
}</string>
                        </property>
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="checkoutPage">
                     <property name="styleSheet">
                      <string notr="true">#checkoutPage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <widget class="QPushButton" name="checkoutButton">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>150</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Checkout</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="holdPage">
                     <property name="styleSheet">
                      <string notr="true">#holdPage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QPushButton" name="holdButton">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>150</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Place Hold</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="holdAvailablePage">
                     <property name="styleSheet">
                      <string notr="true">#holdAvailablePage{
	
	background-color: rgb(227, 238, 219);
}
</string>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <item>
                       <widget class="QPushButton" name="confirmButton">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>150</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Confirm Pickup</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="removeButton">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>150</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Remove Hold</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="footerBackground" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="footerLogo">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>472</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
